src/twisted/web/client.py

```diff
@@ -1456,12 +1456,9 @@ class _StandardEndpointFactory(object):
     """
     def __init__(self, reactor, contextFactory, connectTimeout, bindAddress):
         """
-        @param reactor: A provider of
-            L{twisted.internet.interfaces.IReactorTCP} and
-            L{twisted.internet.interfaces.IReactorSSL} for this L{Agent} to
-            place outgoing connections.
-        @type reactor: L{twisted.internet.interfaces.IReactorTCP} and
-            L{twisted.internet.interfaces.IReactorSSL}
+        @param reactor: A provider to use to create endpoints.
+        @type reactor: L{IReactorTCP} and
```

Comment:
Yes, you are absolutely right.  Thanks for catching this.
Your wording changes are mostly good, except for the `@type`.

In all the places where you specified `@type` can you change to:

```
@type reactor: type of reactor must be same as type used in L{HostnameEndpoint.__init__}
```

That will be correct, and prevent a skew in documentation between client.py and HostnameEndpoint